doc/TESTS.rst.txt

```diff
@@ -136,31 +136,27 @@ modules and develop tests for it.
 Labeling tests
 --------------
 
-As an alternative to ``pytest.mark.<label>``, there are a number of labels you
-can use.
-
 Unlabeled tests like the ones above are run in the default
 ``numpy.test()`` run.  If you want to label your test as slow - and
 therefore reserved for a full ``numpy.test(label='full')`` run, you
-can label it with a decorator::
+can label it with ``pytest.mark.slow``::
 
-  # numpy.testing module includes 'import decorators as dec'
-  from numpy.testing import dec, assert_
+  import pytest
 
-  @dec.slow
+  @pytest.mark.slow
   def test_big(self):
       print 'Big, slow test'
 
 Similarly for methods::
 
   class test_zzz:
-      @dec.slow
+      @pytest.mark.slow
       def test_simple(self):
           assert_(zzz() == 'Hello from zzz')
 
-Available labels are:
+As an alternative to ``pytest.mark.<label>``, there are a number of labels you
+can use:
```

Comment:
More can be done here. We should not promote any of the nose decorators that can be found in `numpy/testing/tests/test_decorators.py`

Reply:
There are pytest decorators avaliable for some of these, e.g., `pytest.mark.skipif`. You can grep current numpy to check if numpy uses any of the decorators.

Reply:
I think all the nose decorators are defined in `numpy/testing/_private/decorators`.

Reply:
Done :+1:

Reply:
> There are pytest decorators avaliable for some of these, e.g., pytest.mark.skipif. You can grep current numpy to check if numpy uses any of the decorators.

I see that there's another section for `pytest.mark.skipif` - https://github.com/numpy/numpy/blob/master/doc/TESTS.rst.txt#known-failures-skipping-tests

I wil try and grep all such labels and list them. And it seems like [this](https://github.com/numpy/numpy/blob/master/doc/TESTS.rst.txt#parametric-tests) needs to be changed too.